pkg/deployments/tasks/deprecated/20230206-erc4626-linear-pool-v3/output/arbitrum.json

```diff
@@ -1,5 +1,5 @@
-{
-  "ERC4626LinearPoolFactory": "0xa3B9515A9c557455BC53F7a535A85219b59e8B2E",
-  "MockERC4626Token": "0x62aaB12865d7281048c337D53a4dde9d770321E6",
-  "MockERC4626LinearPool": "0xf984eb2B8a7eF780245a797A2fCcd82f346409ca"
+{
```

Comment:
Why is there a diff in any of these outputs? Formatting issue? Should just be moved without changes.

Reply:
I think it's a git thing, but there was no reformatting. The files were marked as deleted in one folder and as added in another, and I'm not being able to change it. Still, I just copied the original files, pasted them in the folder, and committed, to make sure no formatting happened, and these files are still being marked as changed